src/Output/Output_L1Error.cpp

```diff
@@ -168,8 +177,16 @@ void Output_L1Error( void (*AnalFunc_Flu)( real fluid[], const double x, const d
          if ( TRank == 0 )
          {
             for (int v=0; v<NERR; v++)
-               fprintf( File[v], "#%*s %*s %*s %*s\n", StrLen_Flt, "Coord.", StrLen_Flt, "Numerical",
-                        StrLen_Flt, "Analytical", StrLen_Flt, "Error" );
+            {
+               fprintf( File[v], "#%*s %c", StrLen_Flt, "Coord. ", coord1 );
+               if ( dim >= 2 )
+               fprintf( File[v], " %*s %c", StrLen_Flt, "Coord. ", coord2 );
+               if ( dim == 3 )
+               fprintf( File[v], " %*s %c", StrLen_Flt, "Coord. ", coord3 );
```

Comment:
Remove the space after "Coord."
```suggestion
               fprintf( File[v], "#%*s %c", StrLen_Flt, "Coord.", coord1 );
               if ( dim >= 2 )
               fprintf( File[v], " %*s %c", StrLen_Flt, "Coord.", coord2 );
               if ( dim == 3 )
               fprintf( File[v], " %*s %c", StrLen_Flt, "Coord.", coord3 );
```